apis/mock/mockEventsApi.js

```diff
@@ -0,0 +1,64 @@
+const Event = require('../classes/Event');
+
+// Mock outputs
+var testNames = [
+    "excite",
+    "object",
+    "look",
+    "itch",
+    "concentrate",
+    "hand",
+    "stamp",
+    "use",
+    "open",
+    "miss",
+    "imagine",
+    "empty"
+]
+
+var testLocations = [
+    "103",
+    "105",
+    "109",
+    "111",
+    "115"
+]
+
+var testSummaries = [
+    "In quis ex ornare, maximus tellus ac, convallis purus.",
+    "Integer sit amet dui ullamcorper, condimentum tellus sed, malesuada dui.",
+    "Nulla aliquam risus vitae tortor vulputate ultricies.",
+    "Quisque in lacus a diam lacinia sollicitudin sit amet volutpat turpis.",
+    "Vivamus semper risus eu ultrices posuere.",
+    "Nam a ex non dui iaculis molestie quis id quam."
+]
+
+function randomItem(array) {
+    var index = Math.floor(Math.random() * array.length);
+    return array[index];
+}
+
+// Generate mock events, start times are the current time offset by a random number of half-hour intervals, end times are an hour from start time
+function generateMockEvents() {
+    var mockEvents = [];
+
+    for (var eventName of testNames) {
+        var name = eventName + "GT";
+
+        var startTimeOffset = (3600000 / 2) * (1 + Math.floor(Math.random() * 7))
+        var startTime = new Date(Date.now() + startTimeOffset);
+        var endTime = new Date(startTime.toString());
+        endTime.setHours(endTime.getHours() + 1);
```

Comment:
Will this break if it's 11pm? Like will it go back a day, or no?

Reply:
Just tested it, it will roll over to the next day